ethers-core/src/types/trace/geth.rs

```diff
@@ -48,6 +48,10 @@ pub struct StructLog {
     pub stack: Option<Vec<U256>>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub storage: Option<BTreeMap<H256, H256>>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub mem_size: Option<i32>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub return_data: Option<String>,
```

Comment:
these need renames because struct is not marked with `camelCase`

Reply:
I thought about it too, but I didn't understand in case of absence of camel_case the rest of the structure is serialized